interfaces/PA-App/src/app/personal-components/create-identity/create-identity.component.ts

```diff
@@ -169,6 +169,9 @@ export class CreateIdentityComponent extends PersonalComponent {
     this.paData.store(this.paData.type.wallet, wallet);
     this.paData.store(this.paData.type.didShort, didShort);
     this.paData.store(this.paData.type.did, did);
+
+    // 9. Store did in user table of PA-account
```

Comment:
Can we also replace the line 171:
```
this.paData.store(this.paData.type.did, did);
```

We would need a `this.paData.getDid()`-method to replace all the existing `paData.retreive('did')` calls.
But it keeps everything more 'together'...

Also, it doesn't need to be the responsibility of the `create-identity`-component to do this. It could be managed by the `PaDataService` as that is already aware of the "offlineStorage"-state etc.

Reply:
We discussed this and decided not to change this